allocations.yml

```diff
@@ -0,0 +1,146 @@
+meta:
+  title: Sid Allocation
+  preamble: |
+    **Mission statement**: a working group to self organize sid ranges for the benefit of users
+
+allocations:
+
+  local:
+    org: local
+    name: Reserved for local use. Put your custom rules in this range.
+    note: Reserved for Local Use - Put your custom rules in this range to avoid conflicts
+    ranges:
+      - start: 1000000
+        size: 1000000
+
+  emergingthreats/open:
+    org: Emerging Threats
+    name: Emerging Threats Open
+    url: https://community.emergingthreats.net/
+    note: "[Emerging Threats Open](https://community.emergingthreats.net/)"
+    ranges:
+      - start: 2000000
+        size: 104000
+      - start: 2400000
+        size: 210000
+    
+  oisf/core:
+    org: OISF
+    name: Suricata Engine Rules
+    note: Suricata Engine Events
+    ranges:
+      - start: 2200000
+        size: 100000
+
+  travisbgreen/hunting:
+    org: Travis Green
+    name: Hunting Ruleset
+    url: https://github.com/travisbgreen/hunting-rules
+    note: "[Hunting Ruleset](https://github.com/travisbgreen/hunting-rules)"
+    ranges:
+      - start: 2610000
+        size: 10000
+
+  3coresec/lateral-movement:
+    org: 3CORESec
+    name: Lateral Movement Ruleset
+    note: "[Lateral Movement Ruleset](https://dtection.io/ruleset)"
+    ranges:
+      - start: 2620000
+        size: 10000
+
+  emergingthreats:
+    org: Emerging Threats
+    name: Emerging Threats
+    note: "[Emerging Threats](https://doc.emergingthreats.net/bin/view/Main/SidAllocation)"
+    ranges:
+      - start: 2700000
+        size: 100000
```

Comment:
This ET entry had no open or pro.. @zoomequipd? Also, the URL?

Reply:
TBD on how this range will get used.

Reply:
Ok, just was curious if it was missing.  But not really a needed item anyways. Just these keys need to be unique.